### PR TITLE
Async/await for mutliple users

### PR DESCRIPTION
**Pull request by MissDove**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -23,21 +25,38 @@ const readInput = () => {
...
-    })
-  )
-    .then(users => {
-      console.log(users);
-      return users;
-    })
-    .catch(error => {
-      console.log(error);
-    });
+// const fetchUsers = usersArray => {
```
Comment (farskid) [inline index.js]: I think it's safe to wipe out this old implementation cause you can always access it on git.

File: index.js [hunk 2]
```diff
@@ -81,18 +100,36 @@ const getButton = document.getElementById(
...
-            getAvatar(user)
-          );
-        });
-      }
-    );
+  async () => {
+    try {
+      flushAvatars();
+      let users = await fetchUsers(readInput());
+      users = users.forEach( user => {
```
Comment (farskid) [inline index.js]: I think we don't need to reassign `users` here cause `forEach` doesn't return anything.
```suggestion
      users.forEach( user => {
        displayAvatar(getAvatar(user));
        });
```

File: index.js [hunk 3]
```diff
@@ -81,18 +100,36 @@ const getButton = document.getElementById(
...
+        displayAvatar(getAvatar(user));
+      });
+      return user;
```
Comment (farskid) [inline index.js]: No need for returning here cause it's the end of the code and no one else is using it's returned value.

**Comment by MissDove** [comment]
Close #2

**Pull request merged**